backend/src/test/java/net/pengcook/user/controller/UserControllerTest.java

```diff
@@ -246,8 +256,39 @@ void report() {
                 .body("reportId", is(1))
                 .body("reporterId", is(9))
                 .body("reporteeId", is(1))
-                .body("reason", is("SPAM"))
-                .body("details", is("스팸 컨텐츠입니다."));
+                .body("reason", is(Reason.SPAM_CONTENT.name()))
+                .body("type", is(Type.RECIPE.name()))
+                .body("targetId", is(1))
+                .body("details", nullValue());
+    }
+
+    @Test
+    @DisplayName("신고 사유 목록을 조회한다.")
+    void getReportReasons() {
+        RestAssured.given(spec).log().all()
+                .filter(document(DEFAULT_RESTDOCS_PATH,
+                        "신고 사유 목록을 조회합니다",
```

Comment:
소소한 부분이기는 한데 다른 메서드들과 마찬가지로 마침표 찍으면 좋을 것 같아요!
```suggestion
                        "신고 사유 목록을 조회합니다.",
```